src/inspector/main_thread_interface.cc

```diff
@@ -255,8 +255,10 @@ void MainThreadInterface::Post(std::unique_ptr<Request> request) {
   if (needs_notify) {
     CHECK_EQ(0, uv_async_send(&main_thread_request_->first));
     if (isolate_ != nullptr && platform_ != nullptr) {
-      platform_->CallOnForegroundThread(isolate_,
-                                        new DispatchMessagesTask(this));
+      std::shared_ptr<v8::TaskRunner> taskrunner =
+        platform_->GetForegroundTaskRunner(isolate_);
+      taskrunner->PostTask(std::unique_ptr<v8::Task>(
+        new DispatchMessagesTask(this)));
```

Comment:
I think we are actually able to use `std::make_unique` for creating unique_ptrs by now, if you prefer

Reply:
Done!